easybuild/easyconfigs/p/protobuf-python/protobuf-python-3.0.2-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'protobuf-python'
+version = '3.0.2'
+versionsuffix= '-Python-%(version)s'
+
+homepage = 'https://github.com/google/protobuf/'
+description = """Python Protocol Buffers runtime library."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/google/protobuf/archive/v%(version)s']
+sources = [SOURCE_TAR_GZ]
+
+versionsuffix = '-Python-%(pyver)s'
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('protobuf', version)
+]
+
+start_dir = 'python'
+
+sanity_check_paths = {
+    'files': ['lib/python%(pyshortver)s/site-packages/protobuf-%(version)s-py%(pyshortver)s.egg/google/__init__.py'],
```

Comment:
@FokkeDijkstra we usually just check for the `lib/python%(pyshortver)s/site-packages` dir, and let checking for the actual `protobuf` Python package via the `import` check that is done by `PythonPackage` (since the `.egg` may be a *file* under certain conditions)